src/main/Extensions/ApplicationSearch/Windows/WindowsApplicationRepository.test.ts

```diff
@@ -105,5 +114,44 @@ describe(WindowsApplicationRepository, () => {
                 ],
                 includeWindowsStoreApps: false,
             }));
+
+        it("should log a warning if a folder path does not exist or is not a directory", () => {
```

Comment:
This test is insufficient, it just tests the behavior of the private method. I'd rather have a test that makes sure that the filtered folders get passed to the powershell script.

Reply:
Sure